apps/ledger-live-desktop/src/renderer/components/Web3AppWebview/index.tsx

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable react/prop-types */
+
+import semver from "semver";
+import { WALLET_API_VERSION } from "@ledgerhq/live-common/wallet-api/constants";
+import React, { forwardRef } from "react";
+import { WalletAPIWebview } from "./WalletAPIWebview";
+import { PlatformAPIWebview } from "./PlatformAPIWebview";
+import TrackPage from "~/renderer/analytics/TrackPage";
+import { WebviewAPI, WebviewProps } from "./types";
+
+export const Web3AppWebview = forwardRef<WebviewAPI, WebviewProps>(
+  ({ manifest, inputs, onStateChange }, ref) => {
+    <TrackPage category="Platform" name="App" appId={manifest.id} params={inputs} />;
```

Comment:
I don't think this will work as you expect, because we don't return it in the react tree.
Also shouldn't we track "Platform" only on the `PlatformAPIWebview` ? or do we consider "Platform" the same as "WalletAPI" for tracking ?

Reply:
What did you assume i'm expecting here ? 😂 

For the events, at this point I don't even know if those events are being used. We'll need to re-evaluate because we are tracking some very random stuff and sometime I feel we are trying to do sentry's job.

For this reason I took the decision to let it as-is

Reply:
Okay because if we actually want that component to work we'll have to use a const and probably use fragment to wrap the webview using the const in the fragment too